app/views/space_contacts/_space_contact.html.erb

```diff
@@ -35,8 +35,8 @@
         <% end %>
       </dl>
     </div>
-    <nav class="transition-opacity opacity-0 group-hover:opacity-100 focus-within:opacity-100">
```

Comment:
Out of scope, but 👍  anyway. We needed it for this trello card: https://trello.com/c/gTemfC5A/183-kan-ikke-redigere-kontaktinformasjon-p%C3%A5-mobil-fordi-den-er-avhengig-av-hover

Reply:
Yeah I wasn't able to delete a space contact from capybara so I think it's fair to remove it until we have a better solution